packages/web/src/pages/artist-coins-launchpad-page/LaunchpadPage.tsx

```diff
@@ -145,18 +161,49 @@ export const LaunchpadPage = () => {
       validateOnMount={true}
       validateOnChange={true}
       onSubmit={(_values: SetupFormValues) => {
-        // Convert coin symbol to uppercase before submission
-        // TODO: Use the processed values in actual submission logic
-        // const finalValues = {
-        //   ...values,
-        //   coinSymbol: values.coinSymbol.toUpperCase()
-        // }
-
-        // For now, this represents completing the entire flow
-        alert('Coin created successfully!') // Temporary success indicator
+        setIsModalOpen(true)
+        if (!user) {
+          throw new Error('No current user found for unknown reason')
+        }
+        if (!connectedWallet) {
+          throw new Error('No connected wallet found')
+        }
+        const parsedPayAmount = _values.payAmount
+          ? Number(new FixedDecimal(_values.payAmount, 9).value) /
+            Math.pow(10, 9)
+          : undefined
+        if (parsedPayAmount !== undefined && isNaN(parsedPayAmount)) {
+          console.error('inititalBuyAudioAmount is not valid', {
+            buyAmountFormValue: _values.payAmount,
+            initialBuyAmountAudioParsed: parsedPayAmount
+          })
+        }
```

Comment:
seems we just log to console but don't handle? should also probably present some UI. ik ur handling error states later but something to note